src/database/database.py

```diff
@@ -35,3 +42,42 @@ def connect():
     except Exception as error:
         logging.fatal(error)
         sys.exit(1)
+
+
+def get_menu_from_db(meal, date):
+    # get menu from database
+    conn = connect()
+    cursor = conn.cursor(cursor_factory=psycopg2.extras.RealDictCursor)
+    cursor.execute(menu_query(meal), (date,))
+    return cursor.fetchone()
+
+
+def get_hidden_cuisines(chat_id):
+    conn = connect()
+    cursor = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+    cursor.execute(settings_query(HIDE_CUISINE), (chat_id,))
+    data = cursor.fetchone()
+
+    if data is None:
+        # insert default settings
+        cursor.execute(settings_insert(), (chat_id, '{}', '{}'))
```

Comment:
just checking, this one don't need `conn.commit()`?

Reply:
need